docs/stable/3-whisper-usage.md

```diff
@@ -24,14 +24,16 @@ title: 3/WHISPER-USAGE
     - [Contact code topic](#contact-code-topic)
     - [Partitioned topic](#partitioned-topic)
     - [Public chats](#public-chats)
-    - [Generic discovery topic](#generic-discovery-topic)
-    - [One-to-one topic](#one-to-one-topic)
+    <!-- - [Generic discovery topic](#generic-discovery-topic) -->
```

Comment:
Why are these commented out?

Reply:
The sections they refer to are also commented out. Looks like whoever commented out the sections left in the links to the sections.

Reply:
I'm moving these into #120